migrations/2_deploy_HealthCare.js

```diff
@@ -1,5 +1,6 @@
 var HealthCare = artifacts.require("./HealthCare.sol");
+const TO_LAB_ADMIN_ADDRESS = "0xF6F2F51c07e44efE7BC25E0EC6B332f39d930ac0";
```

Comment:
Pls add a comment in front of this, something like `hard coded address from Ganache`